src/mcp/tools/core/create_android_app.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { toContent, mcpError } from "../../util.js";
+import { createAndroidApp } from "../../../management/apps.js";
+
+export const create_android_app = tool(
+  {
+    name: "create_android_app",
+    description: "Creates a new Android app in your Firebase project.",
+    inputSchema: z.object({
+      displayName: z.string().optional().describe("The user-friendly display name for your Android app."),
+      packageName: z.string().describe("The package name for your Android app (e.g., com.example.myapp)."),
+    }),
+    annotations: {
+      title: "Create Android App",
+      destructiveHint: false,
+    },
+    _meta: {
+      requiresAuth: true,
+      requiresProject: true,
+    },
+  },
+  async ({ displayName, packageName }, { projectId }) => {
+    try {
+      const androidApp = await createAndroidApp(projectId!, {
+        displayName,
+        packageName,
+      });
+      
+      return toContent(androidApp);
+    } catch (error) {
```

Comment:
This isn't necessary.

Any error throw will be caught and converted to `mcpError` [here](https://github.com/firebase/firebase-tools/blob/master/src/mcp/index.ts#L149)

dito for all tools.

Reply:
Gotcha